<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirForm</class>
 <widget class="QWidget" name="DirForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frameup">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButtonPrev">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/310.bmp</normaloff>
          <disabledoff>:/new/icons/icons/311.bmp</disabledoff>:/new/icons/icons/310.bmp</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonNext">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/312.bmp</normaloff>
          <disabledoff>:/new/icons/icons/313.bmp</disabledoff>:/new/icons/icons/312.bmp</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonUp">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/320.bmp</normaloff>
          <disabledoff>:/new/icons/icons/321.bmp</disabledoff>:/new/icons/icons/320.bmp</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonNew">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/addfolder2.png</normaloff>:/new/icons/icons/addfolder2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxDir"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonBrowse">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/open3.bmp</normaloff>
          <disabledoff>:/new/icons/icons/open4.bmp</disabledoff>:/new/icons/icons/open3.bmp</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addBookmark">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/addbook.png</normaloff>:/new/icons/icons/addbook.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonBookMarkList">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/favourlist.png</normaloff>:/new/icons/icons/favourlist.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameContent">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="framedown">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButtonCut">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/cut.bmp</normaloff>
          <disabledoff>:/new/icons/icons/cut2.bmp</disabledoff>:/new/icons/icons/cut.bmp</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonCopy">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/new/icons/icons/copy.bmp</disabledoff>
          <disabledon>:/new/icons/icons/121.bmp</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonPaste">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/new/icons/icons/paste.bmp</normalon>
          <disabledoff>:/new/icons/icons/paste2.bmp</disabledoff>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonDel">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/delete1.bmp</normaloff>
          <disabledoff>:/new/icons/icons/325.bmp</disabledoff>:/new/icons/icons/delete1.bmp</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonFilter">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/icons/icons/preview1.bmp</normaloff>
          <disabledoff>:/new/icons/icons/preview2.bmp</disabledoff>:/new/icons/icons/preview1.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFilter">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionPrev">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/310.bmp</normaloff>
     <disabledoff>:/new/icons/icons/311.bmp</disabledoff>:/new/icons/icons/310.bmp</iconset>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/312.bmp</normaloff>
     <disabledoff>:/new/icons/icons/313.bmp</disabledoff>:/new/icons/icons/312.bmp</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionUP">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/320.bmp</normaloff>
     <disabledon>:/new/icons/icons/321.bmp</disabledon>:/new/icons/icons/320.bmp</iconset>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAdd_Bookmark">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/addfavour.png</normaloff>:/new/icons/icons/addfavour.png</iconset>
   </property>
   <property name="text">
    <string>Add Bookmark</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionBookmarkList">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/favourlist.png</normaloff>:/new/icons/icons/favourlist.png</iconset>
   </property>
   <property name="text">
    <string>BookmarkList</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCopySelect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/copy.bmp</normaloff>
     <disabledoff>:/new/icons/icons/copy2.bmp</disabledoff>:/new/icons/icons/copy.bmp</iconset>
   </property>
   <property name="text">
    <string>拷贝</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCutSelect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/cut.bmp</normaloff>
     <disabledoff>:/new/icons/icons/cut2.bmp</disabledoff>:/new/icons/icons/cut.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPasteSelect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/paste.bmp</normaloff>
     <disabledoff>:/new/icons/icons/paste2.bmp</disabledoff>:/new/icons/icons/paste.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionMoveToTrash">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/trash2.png</normaloff>:/new/icons/icons/trash2.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>移动到回收站</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
